app/views/admin/loans/_transactions.html.slim

```diff
@@ -10,10 +10,10 @@ section.transactions
         = t('problem_loan_transaction.warning_msg_html', count: txns_with_warnings.count, link: link_to(t('problem_loan_transaction.warning_msg_link', count: txns_with_warnings.count), admin_accounting_problem_loan_transactions_path(loan_id: @loan.id)))
 
   .show-actions.hidden-print
-    - if @add_transaction_available
+    - if policy(@sample_transaction).create?
       = link_to t(".new_transaction"), "#", class: "btn btn-primary",
         data: {action: "new-transaction", project_id: @loan.id}
-  - if @transaction_list_hidden
+  - if @transactions.empty?
```

Comment:
Got rid of the "is qb connected" check for whether or not we show existing transactions. It seems like bad UX to hide existing objects due to connection state. Additionally, admins will be seeing "you can't add txns b/c qb not connected" so they'll have that info. In future we could perhaps add a separate warning for non-admins but that seems like a pretty edgey edge case for now.

Reply:
yes, thank you!